.eslintrc.json

```diff
@@ -13,6 +13,9 @@
     },
     "rules": {
         "no-underscore-dangle":"off",
-        "import/no-extraneous-dependencies":["error", {"devDependencies": false, "optionalDependencies": false, "peerDependencies": false}]
+        "import/no-extraneous-dependencies":["error", {"devDependencies": false, "optionalDependencies": false, "peerDependencies": false}],
+        "consistent-return": "off",
+        "radix": "off",
+        "no-console": "off"
```

Comment:
Why did you add new rules to eslint?

Reply:
Eslint was giving me more unnecessary errors, and I came up with an idea to do add them.